ee/app/license/server/methods.ts

```diff
@@ -1,12 +1,17 @@
 import { Meteor } from 'meteor/meteor';
 import { check } from 'meteor/check';
 
-import { hasLicense } from './license';
+import { hasLicense, getModules } from './license';
 
 Meteor.methods({
-	'license:hasLicense'(feature: string) {
-		check(feature, String);
+	'license:hasLicense'(features: string[]) {
+		check(features, [String]);
 
-		return hasLicense(feature);
+		return features.map((feature) => ({
```

Comment:
we are changing the return here...I couldn't find where we are handling an array instead of a bool in this code... and empty array is 'true'..so,  should we use 'array.every' instead?